src/translation/translations/en.json

```diff
@@ -166,7 +156,6 @@
       "block": "Block",
       "created": "Created",
       "from": "From",
-      "id": "ID",
```

Comment:
Same

Reply:
Same thing here, this was automatically deleted and I couldn't find any place that uses it